wiki/Installation/Linux/en.md

```diff
@@ -0,0 +1,146 @@
+
+2.  Open the terminal and enter as root. Then install `wine`, `winetricks`, `lib32-gnutls`, and `wget`.
+```
+$ su
+# pacman -S wine winetricks lib32-gnutls wget
+# exit
+```
+
+3.  Continue to [Installing osu!](#installing-osu!)
+
+### Ubuntu
+
+1.  Open the terminal. If you are running 64-bit Ubuntu, enter the following commands to enable the `i386` architecture. If not, skip to step 2.
+```
+$ dpkg --add-architecture i386
+$ sudo apt-get update
+```
+
+2.  Add the Wine PPA.
+```
+$ wget 'https://dl.winehq.org/wine-builds/Release.key'
+$ sudo apt-key add Release.key
+$ sudo apt-add-repository 'https://dl.winehq.org/wine-builds/ubuntu/'
+```
+
+3.  Update software and install Wine and Winetricks.
+```
+$ sudo apt-get update
+$ sudo apt-get install wine-devel winetricks
+```
+
+4.  Continue to [Installing osu!](#installing-osu!)
+
+### Fedora
+
+1.  Open the terminal and install the `ia32-libs` equivalent packages.
+```
+$ sudo dnf install alsa-plugins-pulseaudio.i686 glibc-devel.i686 glibc-devel libgcc.i686 libX11-devel.i686 freetype-devel.i686 libXcursor-devel.i686 libXi-devel.i686 libNX_Xext-devel.i686 libXext-devel.i686 libXxf86vm-devel.i686 libXrandr-devel.i686 libXinerama-devel.i686 mesa-libGLU-devel.i686 mesa-libOSMesa-devel.i686 libXrender-devel.i686 libpcap-devel.i686 ncurses-devel.i686 libzip-devel.i686 lcms2-devel.i686 zlib-devel.i686 libv4l-devel.i686 libgphoto2-devel.i686 libcapifax-devel.i686  cups-devel.i686 libxml2-devel.i686 openldap-devel.i686 libxslt-devel.i686 gnutls-devel.i686 libpng-devel.i686 flac-libs.i686 json-c.i686 libICE.i686 libSM.i686 libXtst.i686 libasyncns.i686 libedit.i686 liberation-narrow-fonts.noarch libieee1284.i686 libogg.i686 libsndfile.i686 libuuid.i686 libva.i686 libvorbis.i686 libwayland-client.i686 libwayland-server.i686 llvm-libs.i686 mesa-dri-drivers.i686 mesa-filesystem.i686 mesa-libEGL.i686 mesa-libgbm.i686 nss-mdns.i686 ocl-icd.i686 pulseaudio-libs.i686 python-talloc.x86_64 sane-backends-libs.i686 tcp_wrappers-libs.i686 unixODBC.i686 samba-common-tools.x86_64 samba-libs.x86_64 samba-winbind.x86_64 samba-winbind-clients.x86_64 samba-winbind-modules.x86_64 mesa-libGL-devel.i686 fontconfig-devel.i686 libXcomposite-devel.i686 libtiff-devel.i686 openal-soft-devel.i686 mesa-libOpenCL-devel.i686 opencl-utils-devel.i686 alsa-lib-devel.i686 gsm-devel.i686 libjpeg-turbo-devel.i686 pulseaudio-libs-devel.i686 pulseaudio-libs-devel gtk3-devel.i686 libattr-devel.i686 libva-devel.i686 libexif-devel.i686 libexif.i686 glib2-devel.i686 mpg123-devel.i686 mpg123-devel.x86_64
+```
+
+2.  Install the complier tools.
+```
+$ sudo dnf groupinstall "C Development Tools and Libraries"
+$ sudo dnf groupinstall "Development Tools"
+```
+
+3.  Add the Wine repository. If you are running Fedora 24 or 25, change the `26` in the link to your Fedora version.
+```
+$ dnf config-manager --add-repo 'https://dl.winehq.org/wine-builds/fedora/26/winehq.repo'
+```
+
+4.  Install wine.
+```
+$ dnf install winehq-stable
```

Comment:
Doesn't It need sudo for install btw?

Reply:
I don't recall exactly, I had done this in a VM but it was thrown away a month ago. But looking from the previous steps, it looks like it does... will add it.